tests/uricontent-01/test.yaml

```diff
@@ -0,0 +1,14 @@
+# *** Add configuration here ***
+
+args:
+- -k none
```

Comment:
since the pcap is just a single packet w/o the TCP 3 way handshake (SYN, SYN/ACK, ACK), Suricata will not parse this as HTTP unless you add:
`--set stream.midstream=true`
Then you should be able to add `checks` for an `alert` based on the rules in `test.rules` and a `http` event as well.